keps/sig-cli/3659-kubectl-apply-prune/README.md

```diff
@@ -1,75 +1,5 @@
-<!--
-**Note:** When your KEP is complete, all of these comment blocks should be removed.
```

Comment:
In accordance with this, I removed all the instructional comments for elements we've already completed.